tools/compiler/p/ftops.pl

```diff
@@ -66,15 +66,15 @@ END
 #include "codegen/InstOpCode.hpp"
 #include "p/codegen/PPCOpsDefines.hpp"
 
-const uint32_t OMR::Power::InstOpCode::properties[PPCNumOpCodes] =
+const uint32_t OMR::Power::InstOpCode::properties[NumOpCodes] =
```

Comment:
I thought this file will be deleted. Do we still need it?

Reply:
Yes the last bullet point in the PR description still needs to be complete. I will do this tomorrow.

Reply:
Fixed in a84fdaa